dist/_include-media.scss

```diff
@@ -94,6 +94,89 @@ $unit-intervals: (
   'rem': 0.1
 ) !default;
 
+///
+/// Defines whether support for media queries is available, useful for creating separate stylesheets
+/// for browsers that don't support media queries
+///
+/// @example scss - Disables support for media queries
+///  $im-media-support: false;
```

Comment:
All the `@example` blocks throughout the code currently have the subsequent lines indented by one space. I think this was necessary to make SassDoc display the code blocks correctly, at least in the first version I've used.